apps/changelog-generator/src/Cli.ts

```diff
@@ -1,8 +1,108 @@
-import * as Command from '@effect/cli/Command';
+import { Command, Options } from '@effect/cli';
+import type { ConfigType, DeploymentConfigV2 } from '@blocksense/config-types';
+import {
+  configDir,
+  readConfig,
+  readEvmDeployment,
+} from '@blocksense/config-types';
+import { Console, Effect } from 'effect';
+import Table from 'cli-table3';
+import type { NetworkName } from '@blocksense/base-utils';
+import { parseNetworkName } from '@blocksense/base-utils';
+import { Terminal } from '@effect/platform';
 
-const command = Command.make('hello');
+const listFeeds = Command.make(
+  'list-feeds',
+  {
+    dir: Options.directory('dir').pipe(Options.withDefault(configDir)),
+  },
+  ({ dir }) =>
+    Effect.gen(function* (_) {
+      const network: NetworkName = yield* _(getNetwork);
+      const config = yield* _(
+        Effect.tryPromise(() => readConfig('feeds_config_v2', dir)),
+      );
+      yield* _(printFeeds(config, network));
+    }),
+);
+
+const command = Command.make('changelog-generator').pipe(
+  Command.withSubcommands([listFeeds]),
+);
 
 export const run = Command.run(command, {
-  name: 'Hello World',
+  name: 'Changelog Generator',
   version: '0.0.0',
 });
+
+const printFeeds = (
+  config: ConfigType<'feeds_config_v2'>,
+  network: NetworkName,
+) =>
+  Effect.gen(function* (_) {
+    const deploymentData = yield* _(getDeploymentData(network));
+    if (!deploymentData) {
+      yield* Console.error(
+        `Deployment data for v2 not found for network: '${network}'`,
+      );
+      return;
+    }
+
+    const table = yield* _(initTable());
+
+    config.feeds.forEach(feed => {
+      const contract =
+        deploymentData?.contracts?.CLAggregatorAdapter[Number(feed.id)];
+      table.push([
+        feed.full_name,
+        contract?.address ?? null,
+        contract?.constructorArgs[1] ?? null,
+        contract?.base ?? null,
+        contract?.quote ?? null,
+      ]);
+    });
+    yield* Console.log(table.toString());
+  });
+
+function initTable(): Effect.Effect<Table.Table, never, never> {
+  return Effect.sync(
+    () =>
+      new Table({
```

Comment:
I'd prefer if we could avoid relying on a third-party dependency for this, and instead implement a `drawTable` function in [`@blocksense/base-utils/tty`](https://github.com/blocksense-network/blocksense/blob/main/libs/ts/base-utils/src/tty.ts), similar to [`drawBox`](https://github.com/blocksense-network/blocksense/blob/ae7f8e183c6b1959f4eb00f1112542e79d32a9c9/libs/ts/base-utils/src/tty.ts#L339).